upstairs/src/guest.rs

```diff
@@ -389,9 +389,14 @@ impl Guest {
         rx.wait().await
     }
 
-    pub async fn query_extent_size(&self) -> Result<Block, CrucibleError> {
-        self.send_and_wait(|done| BlockOp::QueryExtentSize { done })
-            .await
+    pub async fn query_extent_info(
```

Comment:
Why is this function implemented under both `impl Guest` and `impl BlockIO for Guest`?  We should be able to just implement it for the latter case (then import the `BlockIO` trait if we need to call it).

(this is a pre-existing condition, but might as well clean it up since we're touching it here)